components/suggestion/switch_channel_provider.jsx

```diff
@@ -716,4 +716,12 @@ export default class SwitchChannelProvider extends Provider {
             component: ConnectedSwitchChannelSuggestion,
         });
     }
+
+    limitChannelsAmount(result, limit = 50) {
```

Comment:
@hmhealey @vovsemenv I think we should put limit here https://github.com/mattermost/mattermost-webapp/blob/a93fda4aa87de490e890225ae09f854d9043bcb1/components/suggestion/switch_channel_provider.jsx#L415 
This is the data we show immediately from locally available results
and https://github.com/mattermost/mattermost-webapp/blob/a93fda4aa87de490e890225ae09f854d9043bcb1/components/suggestion/switch_channel_provider.jsx#L419 here,
And this once we get results from server.

Reply:
In current scenario it will not be 50 items, it may vary based on how many are already there in local results as we remove local from server results here https://github.com/mattermost/mattermost-webapp/blob/a93fda4aa87de490e890225ae09f854d9043bcb1/components/suggestion/switch_channel_provider.jsx#L472